packages/api/src/command/webhook/webhook.ts

```diff
@@ -210,9 +216,16 @@ export const sendPayload = async (
 
 export const sendTestPayload = async (url: string, key: string): Promise<boolean> => {
   const ping = nanoid();
+  const when = dayjs().toISOString();
   const payload: WebhookPingPayload = {
     ping,
+    meta: {
```

Comment:
So I added the ping type, but because we don't store the test webhook, there isn't really any practical ramification of that. Similarly, I added a uuidv4 so that the metadata payload in ping could be of type WebhookMetadataPayload, but since its not being stored their isnt a point of having a uuid. 
thoughts?

Reply:
This is in reference to @leite08 comment and the issue he linked: https://github.com/metriport/metriport-internal/issues/898
Note the second thing mentioned in that issue (removing the type from the mapi wh message had already happened before I believe since it isn't currently present

Reply:
I see your point about not being stored and/or having business utility - but its useful in with the purpose of testing the "protocol"/API contract. So I think this implementation is sound.